webapp/src/components/CalendarPage.tsx

```diff
@@ -1,9 +1,17 @@
+import {
+  Calendar,
+  luxonLocalizer,
+  Event as RBCEvent,
+} from 'react-big-calendar';
 import { Container } from '@mui/material';
+import { DateTime, Settings } from 'luxon';
+import { ProgramWithActivities } from '../types/api';
 import React from 'react';
+import 'react-big-calendar/lib/css/react-big-calendar.css';
 import useApiData from '../helpers/useApiData';
 
-import { ProgramWithActivities } from '../types/api';
-import Program from './Program';
```

Comment:
Also, I think that we're going to need to incorporate a <Program> component on the <CalendarPage> even if we're only showing one Program, we should at least see details about the program.

`Program.tsx` and `Activity.tsx` were added from #427, so you'll need to decide to either adapt the code in those files to work with the code you and Prabh wrote for #428 or to delete those files. There's no sense in keeping them around if they're not helpful for what we need to accomplish #428.